libtenzir/builtins/operators/enumerate.cpp

```diff
@@ -139,18 +139,17 @@ class enumerate_operator2 final : public crtp_operator<enumerate_operator2> {
       for (auto i = int64_t{}; i < detail::narrow<int64_t>(slice.rows()); ++i) {
         check(b->Append(idx++));
       }
-      // TODO: We should insert it at the beginning.
       co_yield assign(selector_, series{int64_type{}, finish(*b)}, slice,
-                      ctrl.diagnostics());
+                      ctrl.diagnostics(), true);
```

Comment:
```suggestion
                      ctrl.diagnostics(), /*prepend =*/ true);
```

Reply:
Done in 4022429192506b504c2c3685a3bd15cbc24f72d6